solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -1065,6 +1073,7 @@ public static class Builder {
     private Long connectionTimeoutMillis;
     private Long requestTimeoutMillis;
     private Integer maxConnectionsPerHost;
+    private Integer maxOutstandingAsyncRequests;
```

Comment:
`maxOutstandingAsyncRequests` is inconsistent with `withMaxOutstandingRequests` (about the async part). I think we can drop the async part, since even synchronous requests use the threshold returned by `getMaxRequestsQueuedPerDestination`.